packages/perseus/src/widgets/__tests__/__snapshots__/interactive-graph.test.ts.snap

```diff
@@ -3250,6 +3250,24 @@ exports[`snapshots should render correctly with locked lines 1`] = `
             <div
               class="default_xu2jcg-o_O-inlineStyles_16155wj"
             >
+              <span
+                style="position: absolute; left: 400px; top: 200px; font-size: 14px; transform: translate(7px, -50%);"
+              >
+                <span
+                  class="mock-TeX"
+                >
+                  x
+                </span>
+              </span>
+              <span
+                style="position: absolute; left: 200px; top: -24px; font-size: 14px; transform: translate(-50%, -7px);"
+              >
+                <span
+                  class="mock-TeX"
+                >
+                  y
+                </span>
+              </span>
```

Comment:
It looks like this addition and the clippath content below are the only actual changes. The other content is still the same, just shifted down to align incorrectly. If there is a way to prevent it from doing that and to only highlight the actual changes, would love to know.

Reply:
Changed the code so the snapshot doesn't have extra changes due to alignment